.github/workflows/ci-kuberenetes-itests.yml

```diff
@@ -0,0 +1,97 @@
+name: Quarkus CI
```

Comment:
should have a unique name (i.e. OpenShift 3.11 CI) otherwise overlap with existing Quarkus CI ?

filename has kubernetes misspelled.